doc/async_worker.md

```diff
@@ -160,31 +159,30 @@ information exposed by the async_hooks API.
 - `[in] resource`: Object associated with the asynchronous operation that
 will be passed to possible async_hooks.
 
-Returns an AsyncWork instance which can later be queued for execution by calling
+Returns a `Napi::AsyncWork` instance which can later be queued for execution by calling
 `Queue`.
```

Comment:
nit: &#96;Queue&#96; should be fully namespaced.

Reply:
Done